core/modules/color/css/color.admin.css

```diff
@@ -28,16 +28,32 @@ input[type=color] {
   margin-top: .5em;
   margin-bottom: .5em;
   clear: both;
-  vertical-align: middle;
 }
 
 /* Color inputs */
+
+/**
+  * Prevent color input labels from being crumbed up together, when there are
+  * some that wrap in multiple lines.
+  */
```

Comment:
These lines have an extra space before the asterisk. They should line up with the first asterisk on the line above, not the second asterisk.

Reply:
Oops 😅 ...fixed! 👍